test/test_wrapper.py

```diff
@@ -870,31 +871,31 @@ def _mime_vs_type(mime, requested_type):
         self.assertEqual(0, _mime_vs_type("text/n3", N3))
         self.assertEqual(0, _mime_vs_type("text/turtle", TURTLE))
         self.assertEqual(0, _mime_vs_type("application/turtle", TURTLE))
-        self.assertEqual(0, _mime_vs_type("application/json", JSON))  # Warning
+        self.assertEqual(0, _mime_vs_type("application/json", JSON))
```

Comment:
I think we should reconsider these asserts, it seems the count is very arbitrary, and I'm not sure that is what matters, maybe we should rather check that the result is actually json rather or something.